cmd/argo/commands/wait.go

```diff
@@ -7,13 +7,14 @@ import (
 	"os"
 	"sync"
 
+	"github.com/argoproj/argo/util"
```

Comment:
This is an internal import, so it's convention to group it together with other internal imports. The convention is:

```
import (

  core

  external

  internal
)
```
Please move this so that is matched with the last group.